sdk/go.mod

```diff
@@ -23,6 +22,7 @@ require (
 	github.com/hashicorp/go-immutable-radix v1.3.1
 	github.com/hashicorp/go-kms-wrapping/entropy/v2 v2.0.0
 	github.com/hashicorp/go-kms-wrapping/v2 v2.0.8
+	github.com/hashicorp/go-metrics v0.5.3
```

Comment:
Moved SDK dependency `armon/go-metrics` to `hashicorp/go-metrics` and bumped `0.4.1` => `0.5.3`